python/polycap.pyx

```diff
@@ -306,6 +306,8 @@ cdef class Photon:
         set_exception(error)
         if rv == -1:
             return None
+        if rv == 0:
+            return None
```

Comment:
Niet zeker of je dit de meest elegante manier vindt...
Dus nu is -1 een error, 0 dat photon polycapillair einde niet haalde (dus weights zeer klein)
Mss is het voor correctheid op zich wel beter om ook bij `rv == 0` toch de weights door te geven? Indien het geval dient de `python.py` tests wat aangepast te worden...

Reply:
>Zijn de weights nuttig als het einde van het polycapillair toch niet bereikt is? Anders mag het gewoon `None` blijven van mij.

Hm, goed punt. Idd niet nuttig. Ik was aan het redeneren dat de gewichten dan gewoon heel klein waren, maar in se wel bruikbaar. Maar inderdaad stopt de berekening op dat punt direct, dus het zou fout zijn om met die foton verder te rekenen of wat dan ook.

Laten we het dan zo laten :)
Mag dan gemerged worden voor mijn part.

Reply:
Kan je nog die check in `polycap.pyx` voor `rv == -1` eruit halen? Die lijn kan toch niet bereikt worden aangezien de error zal getransformeerd worden in een exception door `set_exception`.

Nieuwe commit niet nodig: gewoon de laatste amenderen en force pushen.